src/main/resources/digital/slovensko/autogram/build.properties

```diff
@@ -1,7 +1,8 @@
-name=Autogram
```

Comment:
Tu si nie som uplne isty ci to globalne chceme premenovat. Kde vsade to bude potom malym?